### ISSUE TITLE
Error thrown on refresh

### ISSUE THREAD

**Issue by JohnClema** [mheap/trello-cli#50]
After calling `trello refresh` from a global npm installation on a MacBook Pro (Retina, 15-inch, Mid 2015) High Sierra 10.13.4 I received the following error:

``` shell 
Organization, board, list, and user cache refreshed
/usr/local/lib/node_modules/trello-cli/src/translator.js:114
            throw err;
            ^

Error: [object Object]
    at Request._callback (/usr/local/lib/node_modules/trello-cli/node_modules/node-trello/lib/node-trello.js:88:13)
    at Request.self.callback (/usr/local/lib/node_modules/trello-cli/node_modules/request/request.js:187:22)
    at Request.emit (events.js:180:13)
    at Request.<anonymous> (/usr/local/lib/node_modules/trello-cli/node_modules/request/request.js:1044:10)
    at Request.emit (events.js:180:13)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/trello-cli/node_modules/request/request.js:965:12)
    at IncomingMessage.emit (events.js:185:15)
    at endReadableNT (_stream_readable.js:1106:12)
    at process._tickCallback (internal/process/next_tick.js:114:19)
```

The refresh still appeared to work correctly despite the error.

**Comment by schadilibre** [comment]
I have the same error with refresh, also on Mac OSX (Mojave)

I'm wondering if the problem is not linked with character encoding : 
I've have the same error when using **show-cards** on board with french accentuated chars (for instance `â` char), but no problem with other boards...

**Comment by kevinawoo** [comment]
Mine seems to be a rate limit issue
```
...
  statusCode: 429,
  responseBody:
   { error: 'RATE_LIMIT_EXCEEDED',
     message: 'Exceeded rate limit to /1/member' },
  statusMessage: 'Too Many Requests' }
```

**Comment by tobybatch** [comment]
Me too.  I can see cards but ```trello refresh``` throws an error:
```
Organization, board, list, and user cache refreshed
/home/tobias/workspace/trello-cli/src/translator.js:114
            throw err;
            ^

Error: [object Object]
    at Request._callback (/home/tobias/workspace/trello-cli/node_modules/node-trello/lib/node-trello.js:88:13)
    at Request.self.callback (/home/tobias/workspace/trello-cli/node_modules/request/request.js:187:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/home/tobias/workspace/trello-cli/node_modules/request/request.js:1044:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/home/tobias/workspace/trello-cli/node_modules/request/request.js:965:12)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
```

**Comment by mheap** [comment]
It does look like it's a rate limit error. I'll try and take a look

**Comment by baaart** [comment]
Same here, I also do not see any of my boards, i assume it is because i cant refresh? Do you have any sollution for this problem maybe?

**Comment by mheap** [comment]
I've picked this up again and can't seem to get to the bottom of it. I'm respecting rate limits as documented on https://help.trello.com/article/838-api-rate-limits on my local branch and still getting rate limited, despite the headers saying that I'm fine

```
X-RATE-LIMIT-API-KEY-INTERVAL-MS: 10000
X-RATE-LIMIT-API-KEY-MAX: 300
X-RATE-LIMIT-API-KEY-REMAINING: 299
X-RATE-LIMIT-API-TOKEN-INTERVAL-MS: 10000
X-RATE-LIMIT-API-TOKEN-MAX: 100
X-RATE-LIMIT-API-TOKEN-REMAINING: 99
X-RATE-LIMIT-MEMBER-INTERVAL-MS: 10000
X-RATE-LIMIT-MEMBER-MAX: 200
X-RATE-LIMIT-MEMBER-REMAINING: 199
```

I've raised a support ticket and will see what they suggest

**Comment by baaart** [comment]
hi sorry, my bad, my error is slightly different, concerning self signed cert

```
.nvm/versions/node/v10.4.0/lib/node_modules/trello-cli/src/translator.js:170
          throw err;
          ^

Error: self signed certificate in certificate chain
    at TLSSocket.onConnectSecure (_tls_wrap.js:1055:34)
    at TLSSocket.emit (events.js:182:13)
    at TLSSocket._finishInit (_tls_wrap.js:635:8)
```

Do you know what is the issue here?

**Comment by mheap** [comment]
Got a response!

> Thanks for writing in, and sorry for the trouble. The /1/members route has special rate-limiting rules, which you may be running into. These are not documented in that help article at the moment, but check out this page for more info: https://developers.trello.com/docs/rate-limits#section-special-route-limits.

> I think the best bet for your use case would be to use the boards route, with the members=all parameter. Check out these links for more info: https://developers.trello.com/reference#boardsboardid-1 and https://developers.trello.com/reference#members-nested-resource. This would enable you to fetch all the members for your board in a single request.

I'll try and find time to fix the issue now that there's a proposed solution

**Comment by mheap** [comment]
@baaart That looks like an SSL issue. Could you raise another issue?

**Comment by mheap** [comment]
It's been a long time coming, but I just fixed this issue! `trello-cli` version `0.3.0` is now available via `npm`

**Comment by JohnClema** [comment]
Legend! Thanks @mheap
